webui/ocw/views.py

```diff
@@ -45,6 +47,12 @@ def update_status(request):
     return redirect('instances')
 
 
+def cron(request):
+    update(request)
```

Comment:
don't think it is good idea we have separate call for that , let's leave it like this